js/hotgraphicModel.js

```diff
@@ -0,0 +1,48 @@
+define([
+    'core/js/adapt',
+    'core/js/models/itemsModel'
+], function(Adapt, ItemsModel) {
+
+    var HotgraphicModel = ItemsModel.extend({
+
+        defaults: function() {
+            return _.extend({}, _.result(ItemsModel.prototype, "defaults"), {
+                _isPopupOpen: false,
+                _marginDir: (Adapt.config.get('_defaultDirection') == 'rtl') ? 'right': 'left'
+            });
+        },
+
+        initialize: function() {
+            this.set('_itemCount', this.get('_items').length);
+
+            ItemsModel.prototype.initialize.apply(this, arguments);
+        },
+
+        prepareHotgraphicModel: function() {
+            this.resetActiveItems(false);
+            this.set('_isPopupOpen', false);
+            this.set('_component', 'hotgraphic');
+            this.set('body', this.get('originalBody'));
+            this.set('instruction', this.get('originalInstruction'));
+            
+            return this;
+        },
+
+        checkCompletionStatus: function() {
```

Comment:
I think it would be good practice to add a comment here flagging that this overrides the function in ItemsModel, with a brief explanation of why that's necessary i.e. to allow HotGraphic to support `inview` completion as well as `allItems`